Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -64,15 +64,20 @@ open class Core: UIBaseObject, UIGestureRecognizerDelegate {
         backgroundColor = UIColor.black
 
         #if os(iOS)
-        mediaControl = loader.mediaControl.create()
+        if let externalMediaControl = options[kMediaControl] as? MediaControl.Type {
+            mediaControl = externalMediaControl.create()
+        } else {
+            mediaControl = MediaControl.create()
+        }
         mediaControl?.core = self
+        
         addTapRecognizer()
         #endif
 
         bindEventListeners()
-        loadPlugins(loader)
+        Loader.shared.loadPlugins(in: self)
```

Comment:
Like in the `Container` constructor, it would be better to isolate plugin loading calls into a separate component.